### PR TITLE
Added React Router and linked Navbar to placeholder pages

### LINKED ISSUE #22: Setup React Router
This allows for page navigation

### PR DESCRIPTION
**Pull request by joyx9**
This PR solves #22 and adds React Router to the app. The Navbar is now able to navigate to the About, Opening Info, and Custom Notation pages. 

### Future TODO

- The other pages besides the Home Page are currently placeholders - will add relevant text, information, and styling to these pages

### CODE REVIEW

**Review by austincho — APPROVED** [review]
LGTM. Just make sure you push it into main instead of project_3!

File: client/src/components/navigation/Navbar.js [hunk 1]
```diff
@@ -17,28 +18,25 @@ const useStyles = makeStyles({
 });
 
 // TODO: Add functionality for page changes when react-router is implemented
-const Navbar = () => {
+const Navbar = (props) => {
```
Comment (austincho) [inline client/src/components/navigation/Navbar.js:21]: Just curious is NavBar being used anywhere? I noticed we removed it in App.js

Comment (joyx9) [reply client/src/components/navigation/Navbar.js:21]: It's on line 26 in App.js!

Comment (austincho) [reply client/src/components/navigation/Navbar.js:21]: ah lol thanks!

**Comment by valarao** [comment]
Good thing we have CI haha, see the logs

https://github.com/valarao/chocolate-thunder-chess/pull/58/checks?check_run_id=2834865310#step:6:31

**Comment by joyx9** [comment]
Whew, I think everything looks good now but if anyone wants to put in a review let me know if there is any trouble, otherwise will be merging in tonight :D

**Comment by valarao** [comment]
LGTM!

**Pull request merged**